Makefile

```diff
@@ -0,0 +1,33 @@
+PREFIX ?= /usr/local
+BASH_COMPLETION_DIR := $(shell pkg-config --variable=completionsdir bash-completion) 
```

Comment:
Not sure why I didn't spot this before, sorry. If I don't have `bash-completion` installed and findable with `pkg-config` (e.g. on OSX) then this all fails here with:

```
Package bash-completion was not found in the pkg-config search path.
Perhaps you should add the directory containing `bash-completion.pc'
to the PKG_CONFIG_PATH environment variable
No package 'bash-completion' found
make: *** [install] Error 127
```